### PR TITLE
[Admin] Filter by taxons on products grid

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | -
| License         | MIT

screen shot 2016-10-21 at 15 13 20

### CODE REVIEW

File: features/product/managing_products/browsing_products_by_taxon.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        Given the store is available in "English (United States)"
+        And the store classifies its products as "Mugs" and "Pugs"
+        And the store has a product "Colorful mug"
+        And this product belongs to "Mugs"
+        And the store has a product "Young pug"
+        And this product belongs to "Pugs"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Browsing defined products
```
Comment (michalmarcinkowski) [inline features/product/managing_products/browsing_products_by_taxon.feature]: Viewing only products from specified taxon

File: features/product/managing_products/browsing_products_by_taxon.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        Given I am browsing products
+        When I filter them by "Pugs" taxon
+        Then I should see a product with name "Young pug"
+        But I should not see any product with name "Colorful mug"
```
Comment (michalmarcinkowski) [inline features/product/managing_products/browsing_products_by_taxon.feature]: We should add one more scenario `Viewing all products` and verify we see products from all categories as well as unclassified products.

File: src/Sylius/Bundle/AdminBundle/Resources/config/routing/product.yml [hunk 3]
```diff
@@ -16,6 +16,36 @@ sylius_admin_product:
...
+sylius_admin_product_index:
+    path: /products/
+    methods: [GET]
+    defaults:
+        _controller: sylius.controller.product:indexAction
+        _sylius:
+            section: admin
+            grid: sylius_admin_product
+            template: SyliusAdminBundle:Product:index.html.twig
+            permission: true
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AdminBundle/Resources/config/routing/product.yml]: Redundant.

Same above.

**Comment by pjedrzejewski** [comment]
Thank you Kamil!

**Pull request merged**